bqetl_project.yaml

```diff
@@ -387,6 +387,18 @@ generate:
   stable_views:
     skip_datasets:
     - mlhackweek_search
+  usage_reporting:
+    apps:
+      fenix:
+        channels:
+        - nightly
+      firefox_ios:
+        channels:
+        - nightly
+      # firefox_desktop:
```

Comment:
Is the `firefox_desktop` commented out for a reason?
I see the usage of `firefox_desktop` in [clients_daily](https://github.com/mozilla/bigquery-etl/pull/6858/files#diff-37ce5183fad2c4f79538026a3f0d5016dcc64acc1cf7c63767a2b7bd574d4cd4R21)